qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -1469,6 +1468,320 @@ wait_for_image_in_omap()
     wait_for_omap_keys ${cluster} ${pool} rbd_mirror_leader image_map
 }
 
+create_group()
+{
+    local cluster=$1
+    local pool=$2
+    local group=$3
+
+    rbd --cluster ${cluster} group create ${pool}/${group}
+}
+
+rename_group()
+{
+    local cluster=$1
+    local pool=$2
+    local group=$3
+    local new_name=$4
+
+    rbd --cluster ${cluster} group rename ${pool}/${group} ${pool}/${new_name}
+}
+
+remove_group()
+{
+    local cluster=$1
+    local pool=$2
+    local group=$3
+
+    rbd --cluster ${cluster} group remove ${pool}/${group}
+}
+
+group_add_image()
+{
+    local cluster=$1
+    local pool=$2
+    local group=$3
+    local image_pool=$2
+    local image=$3
+
+    rbd --cluster ${cluster} \
+        group image add ${pool}/${group} ${image_pool}/${image}
+}
+
+group_remove_image()
+{
```

Comment:
personally I would just make them generalized :-)

But, no objections if we want to care on the test arg names.

Reply:
In case my previous comment was unclear:

    local cluster=$1
    local pool=$2
    local group=$3
    local image_pool=$2
    local image=$3

We are reusing $2 and $3 for both group and image pools and group and image names. It is better to have separate args 
    local cluster=$1
    local group_pool=$2
    local group=$3
    local image_pool=$4
    local image=$5